### DISCUSSION TITLE
All pages have "Unknown page name" under "Speed Insights"

### DISCUSSION THREAD

**Discussion by Svish** [vercel/community#2864] [Help]
### Summary

I have enabled Speed Insights on my Next.js site hosted on Vercel. Had a look today, and found something weird:

![image](https://github.com/vercel/community/assets/142162/c2997582-95a7-43bf-9d74-58ad8642afb4)

Why does apparently none of my pages have a page name? What _is_ a page name? They all definitely have titles in the browser, so page name seems to be something else? But, what, exactly?

### Steps to Reproduce

1. Enable "Speed Insights"
2. Deploy Next.js app
3. ...
4. Get "Unknown page name" on all pages? 🤷‍♂️

**Comment by amyegan** [comment]
Hi @Svish. I just tried it with my test site running Next 13, and I see Page Names based on their path (similar to their URLs).

![image](https://github.com/vercel/community/assets/8247105/cdd23b64-62c5-4fcd-8300-883768fdd5e7)

If you still see this problem, please share your public repo or [a minimal reproduction](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) to help me get a better look at what's going wrong.

**Reply by Svish** [reply]
Still seeing the problem, yeah. Made the repo public here:  
https://github.com/Svish/geekality.net

It's the repo that's connected to Vercel and what's deployed currently.

**Reply by wouterraateland** [reply]
Hi @Svish did you manage to solve this?
I have the same problem with a nextjs@13 app which also uses standard Vercel deploy etc.

**Comment by Svish** [comment]
Nope 🤷‍♂️
